### PR TITLE
Add README.adoc

### PR DESCRIPTION
**Pull request by mtyazici**

### CODE REVIEW

**Review by mesutcelik — APPROVED** [review]

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+== Prerequisites
+
+* ~10 mins
+* Docker and Docker Compose
+* Apache Maven 3.2+
+
+== The Quarkus Application Structure
+
+This guide showcases how to setup a basic Quarkus application to work with Hazelcast Client-Server Topology.
```
Comment (pivovarit) [inline README.adoc]:
```suggestion
This guide showcases how to set up a basic Quarkus application to work with Hazelcast Client-Server topology. 
```

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+This guide showcases how to setup a basic Quarkus application to work with Hazelcast Client-Server Topology. 
+
+The `put` operation places a key-value pair to Hazelcast and `get` operation returns the value along with the Container Name.
+
+The Container Name is present to make it clear from which instance the value is returned from.
+
+== Using Hazelcast Client in the Application
+
+In `pom.xml` we add quarkus-hazelcast-client dependency as follows:
```
Comment (pivovarit) [inline README.adoc]: In `pom.xml` we add _quarkus-hazelcast-client_ dependency as follows:

**Review by pivovarit — APPROVED** [review]
Looks good to me - added a few suggestions. It was the Kubernetes guide that required a bit more work.

File: README.adoc (hunk 2 above)
Comment (mtyazici) [inline README.adoc]:
```suggestion
In `pom.xml` we add _quarkus-hazelcast-client_ dependency as follows:
```

**Pull request merged**